pilota-build/test_data/thrift_workspace_with_split/output/article/src/gen.rs

```diff
@@ -0,0 +1,32 @@
+pub mod gen {
+    #![allow(warnings, clippy::all)]
+
+    pub mod article {
```

Comment:
I think every mod should be placed under a new directory as Rust does in the original design to avoid file name conflict and too many files under one directory.